### PR TITLE
Create 19.0.0.xml

### PR DESCRIPTION
**Pull request by jospoortvliet**
WIP! Adding test.

Signed-off-by: Jos Poortvliet <jospoortvliet@gmail.com>

### CODE REVIEW

File: data/19.0.0.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<release xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../schema.xsd" version="18.0.0">
+  <changelog href="https://nextcloud.com/changelog/#19-0-0"/>
+  <whatsNew lang="en">
+    <regular>
+      <item>Talk 9 integrates document viewing/editing and a grid view</item>
```
Comment (nickvergessen) [inline data/19.0.0.xml:6]: Users can't enable talk, or talk will tell it themselves, so this should be moved to admin?

File: data/19.0.0.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      <item>Deck integration in Calendar - easily track your deadlines!</item>
+      <item>Assign groups to guest accounts you create</item>
```
Comment (nickvergessen) [inline data/19.0.0.xml:8]: admin only

File: data/19.0.0.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<release xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../schema.xsd" version="18.0.0">
+  <changelog href="https://nextcloud.com/changelog/#19-0-0"/>
+  <whatsNew lang="en">
+    <regular>
+      <item>Talk 9 integrates document viewing/editing and a grid view</item>
+      <item>Deck integration in Calendar - easily track your deadlines!</item>
```
Comment (nickvergessen) [inline data/19.0.0.xml:7]: When not enabled the users can't do anything and don't even know what deck might be about?

File: data/19.0.0.xml (hunk 1 above)
Comment (jospoortvliet) [reply data/19.0.0.xml:6]: Well, if users don't have Talk installed, they can ask the sysadmin... I'm happy to advertise Talk a little this way. You disagree?

File: data/19.0.0.xml (hunk 2 above)
Comment (jospoortvliet) [reply data/19.0.0.xml:8]: aaah that should go to admin then somehow. Let me edit it.

File: data/19.0.0.xml (hunk 3 above)
Comment (jospoortvliet) [reply data/19.0.0.xml:7]: Same as with Talk - happy to advertise it, they can complain to the admin that they want it :rocket:

Comment (juliushaertl) [reply data/19.0.0.xml:7]: That is actually not released in deck yet.

Comment (blizzz) [reply data/19.0.0.xml:7]: But Talk is tightly bound to Nextcloud and installed on fresh setups. Not so Deck.

> That is actually not released in deck yet.

And this nails it :)

Comment (jospoortvliet) [reply data/19.0.0.xml]: Removed.

**Pull request merged**